photon.json

```diff
@@ -18,7 +18,7 @@
       ],
       "boot_command": [
         "<esc><wait>",
-        "vmlinuz initrd=initrd.img root=/dev/ram0 loglevel=3 ks=http://{{ .HTTPIP }}:{{ .HTTPPort }}/photon-kickstart.json photon.media=cdrom",
+        "vmlinuz initrd=initrd.img root=/dev/ram0 loglevel=3 ks=http://{{ .HTTPIP }}:{{ .HTTPPort }}/photon-kickstart.json photon.media=cdrom insecure_installation=1",
```

Comment:
What is `insecure_installation` doing? Asking in terms of security-relevant requirements for releasing appliances/Flings.

Reply:
The `insecure_installation` is now a mandatory parameter in PH4 if your kickstart endpoint is non-HTTPs. We use Packer to automate the build and when it spins up the webserver to host KS, it is serve over HTTP only and hence this is needed